kafka-demos/kafka-connect-pipeline/README.md

```diff
@@ -38,7 +38,8 @@
 
 * Check topic contents
 
-      kafka-avro-console-consumer --bootstrap-server localhost:9092 --topic purchases --from-beginning --property print.key=false
+      docker-compose exec connect bash
```

Comment:
People may have docker compose as a docker plugin. I think `docker-compose` is deprecated now and we should use `docker compose` (two words instead of dash inbetween).